core/src/main/resources/reference.conf

```diff
@@ -102,6 +102,32 @@ datastax-java-driver {
   # and metrics.
   // session-name = my_session
 
+  # A set of customizable actions that are performed when the driver starts up, i.e.,
+  # each time a new session is initialized.
+  # Actions in this section that involve printing log messages will always use the special logger
+  # 'com.datastax.oss.driver.api.core.Startup' at level INFO.
+  startup {
+
+    # Whether or not to print basic information about the initialization process,
+    # such as the driver name, its version number, and the process PID, if available.
+    print-basic-info = true
+
+    # An optional custom banner to be printed at startup. This can be useful to differentiate
+    # between two applications using the driver.
+    #
+    # Tip 1: this option works best with multi-line strings, as in the example below.
+    #
+    # Tip 2: you can include system properties, environment variables and/or references to any
+    # other configuration option, as in the example below. Beware that any reference must point
+    # to an existing value, otherwise an error will be raised.
+    //custom-banner = """
```

Comment:
This really shows all the potential of TypeSafe Config imo.

Reply:
👍 Nice, especially considering that in a typical deployment this can be changed without recompiling.